spec/integration/jeff-aboutus.spec.js

```diff
@@ -2,46 +2,46 @@
 
 /* global casper */
 
-casper.test.begin('About Us - Navigation Test', 16, function suite(test){
-  casper.start('http://localhost/neukinnis/client/aboutus.html', function(){
+casper.test.begin('About Us - Navigation Test', function suite(test){
+  casper.start('http://localhost:3000/aboutus.html', function(){
     test.assertTitle('About Us', 'Finds page title ' + this.getTitle());
-    test.assertExists('a[href="index.html"]', 'Checks existence of link to Home page');
+    test.assertExists('a[href="/index.html"]', 'Checks existence of link to Home page');
   });
   casper.then(function(){
-    this.click('a[href="index.html"]', 'Navigates and clicks on link to Home page');
+    this.click('a[href="/index.html"]', 'Navigates and clicks on link to Home page');
   });
   casper.then(function(){
     test.assertTitle('Neukinnis', 'Finds page title ' + this.getTitle());
     this.back();
   });
+  // casper.then(function(){
```

Comment:
I don't like seeing commented-out code.  If you don't want code, I think it should be deleted because we can always fetch code that we deleted in the past by digging up a git commit.